kids-learning-app/assets/css/styles.css

```diff
@@ -0,0 +1,178 @@
+}
+
+body{
+    background-color: #000000;
+}
+
+/* header styles */
+header{
+    display: flex;
+    justify-content: space-between;
+    background-color: #B5DF91;
+    padding: 20px 40px;
+    font-size: 20px;
+    font-weight: 600;
+}
+
+/* body made flex for auto align */
+.body-flex{
+    width: 80%;
+    margin: 175px auto;
+    display: flex;
+    flex-direction: column;
+    row-gap: 70px;
+    justify-content: center;
+    align-items: center;
+}
+
+.title{
+    color: #ffffff;
+    font-size: 40px;
+    font-weight: 600;
+}
+
+/*  contains the shapes or shape buttons */
+.shapes-container{
+    width: 600px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+
+/* font-awesome icons tick mark */
+.fa-solid.fa-check{
+    position: relative;
+    bottom: -50px;
+    left: 50px;
+    color: white;
+    font-size: 50px;
+    /* display: none; */
+}
+
+/* icon edited for triangle alone */
+.fa-solid.fa-check.chktriangle{
+    left: -25px;
+    bottom:-60px;
+}
+
+.shapes-container div:hover{
+    cursor: pointer;
+}
+
+/* shapes button */
+.circle{
+    width:150px;
+    height: 150px;
+    background-color: #43967e;
+    border: solid #43967E;
+    border-radius: 50%;
+}
+
+.triangle {
+    width: 0;
+    height: 0;
+    border-left: 75px solid transparent;
+    border-right: 75px solid transparent;
+    border-bottom: 150px solid #F3AFDA;
+}
+
+.square{
+    width: 150px;
+    height: 150px;
+    background-color: #E0E0E0;
+    border: solid #E0E0E0;
+}
+
+button:hover{
+    cursor:pointer;
+}
+
+.nextButton{
+    width: 300px;
+    height: 60px;
+    background-color: #000000;
+    border: 2px solid #ffffff;
+    border-radius: 5px;
+    color: #ffffff;
+    font-size: 20px;
+    font-weight: 600;
+}
+
+.dimension{
+    height:70px;
+    width: 600px;
+    background-color: #000000;
+    border: 2px solid #ffffff;
+    color: #ffffff;
+    padding-left: 15px;
+    font-size: 20px;
+}
+
+input::placeholder{
+    padding-left: 15px;
+    color: #E0E0E0;
+    opacity: 1;
+}
+
+
+/* table for results */
+.table{
+    padding: 0;
+}
+
+.row{
+    width: 500px;
+    height: 50px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.col1,.col2,.col3{
+    display: inline-block;
+    height: 50px;
+}
+
+.col1{
+    width: 200px;
+    margin: 0;
+    padding-top: 15px;
+    padding-left: 10px;
+    background-color: #2b60ce;
+    color: #ffffff;
+    text-align: left;
+    font-size: 15px;
+}
+
+.col2{
+    width:150px;
+    margin: 0;
+    padding-top: 15px;
+    padding-left: 10px;
+    background-color: #4a5d7c;
+    color: #ffffff;
+    text-align: left;
+    font-size: 15px;
+}
+
+.col3{
+    width:150px;
+    margin: 0;
+    padding-top: 15px;
+    padding-left: 10px;
+    background-color: rgb(75,134,247);
+    color: #ffffff;
+    text-align: left;
+    font-size: 15px;
+}
```

Comment:
There are few redundant CSS properties with the same value, think of a solution that can prevent this